docs/policies.rst

```diff
@@ -393,5 +394,78 @@ It is recommended to use
           ``mu_neg = mu_pos`` and ``use_max_sim_neg = False``. See
           `starspace paper <https://arxiv.org/abs/1709.03856>`_ for details.
 
+Two Stage Fallback Policy
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This policy handles low NLU confidence in multiple stages.
+
+- If a NLU prediction has a low confidence score, the user is asked to confirm
+  whether they really had this intent.
+
+    - If they confirm, the story continues as if the intent was recognized
+      with high confidence from the beginning.
+    - If they deny, the user is asked to restate his intent.
+
+- Clarification
+
+    - If the recognition of the clarification was confident, the story
+      continues as if the user had this intent from the beginning.
+    - If the clarification was not recognised with high confidence, the user
+      is asked to confirm the recognized intent.
+
+- Second confirmation
+
+    - If the user confirms the intent, the story continues as if the user had
+      this intent from the beginning.
+    - If the user denies, an ultimate fallback action is triggered
+      (e.g. a handoff to a human).
+
+Configuration
+"""""""""""""
+
+To use this policy, include the following in your policy configuration.
+Note that you cannot use this together with the default fallback policy.
+
+.. code-block:: yaml
+
+    policies:
+      - name: TwoStageFallbackPolicy
+        nlu_threshold: 0.3
+        core_threshold: 0.3
+        fallback_action_name: "action_default_fallback"
+
++-------------------------------+---------------------------------------------+
+| ``nlu_threshold``             | Min confidence needed to accept an NLU      |
+|                               | prediction                                  |
++-------------------------------+---------------------------------------------+
+| ``core_threshold``            | Min confidence needed to accept an action   |
+|                               | prediction from Rasa Core                   |
++-------------------------------+---------------------------------------------+
+| ``fallback_action_name``      | Name of the action to be called if the      |
+|                               | confidence of intent / action prediction    |
+|                               | is below the threshold                      |
++-------------------------------+---------------------------------------------+
+
+.. note::
+
+    It is required to have the two intents ``confirm`` and ``deny`` in the
+    domain of the bot. These are used to identify whether the user confirmed
+    to a suggestion or denied it.
```

Comment:
domain of the bot, to determine whether the user confirms or denies a suggestion.